commons/pom.xml

```diff
@@ -14,7 +14,7 @@
     <properties>
         <!-- The commons build sometimes fails on Travis with Coverage checks have not been met; -->
         <!-- I haven't been able to figure out why -->
-        <jacoco-percentage>0.6</jacoco-percentage>
+        <jacoco-percentage>0.4</jacoco-percentage>
```

Comment:
Why is it downgraded?

Reply:
agreed. I missed some changes on my machine